test/core/end2end/generate_tests.bzl

```diff
@@ -15,7 +15,7 @@
 
 """Generates the appropriate build.json data for all the end2end tests."""
 
-load("//bazel:grpc_build_system.bzl", "grpc_cc_binary", "grpc_cc_library")
+load("//bazel:grpc_build_system.bzl", "grpc_cc_binary", "grpc_cc_library", "is_msvc")
```

Comment:
Actually, you may need to create a placeholder for this "is_msvc" too in our other version of "grpc_build_system.bzl".